doc/refactorings.md

```diff
@@ -893,6 +894,53 @@ class Hello
 }
 ```
 
+Expand Class
+------------
+
+Expand the class name from unqualified name to fully qualified name.
+
+- **Command**: _VIM function only_
+- **VIM plugin**: _VIM function only_
+- **VIM function**:`:call phpactor#ClassExpand()`
+
+### Motivation
+
+Although `use` statement make code cleaner, but sometimes the code would be more
+readable if we specify the full qualified name. For example, we might register
+a list of listeners in a file.
```

Comment:
=> Although importing classes can make code cleaner, sometimes the code can be more readable if the fully qualified name is specified.

Reply:
Does that make sense?

Reply:
That sounds fine, I'll take it.
Let me ask that if the **"but"** kind of weird in there so you trying to drop it? (Sorry, I'm not so good at English)

Reply:
Yes, it's a bit weird -- it would work with `Blah statements make the code cleaner, but sometimes`. I guess it's either `although` or `but`. (disclaimer I'm a native speaker but not a language expert, so cannot say _why_, just it feels better :) )